temporalio/workflow.py

```diff
@@ -244,6 +245,17 @@ def signal(
     *,
     dynamic: Literal[True],
     unfinished_policy: HandlerUnfinishedPolicy = HandlerUnfinishedPolicy.WARN_AND_ABANDON,
+    description: Optional[str] = None,
+) -> Callable[
+    [CallableSyncOrAsyncReturnNoneType], CallableSyncOrAsyncReturnNoneType
+]: ...
+
+
+@overload
+def signal(
+    *,
+    description: str,
+    unfinished_policy: HandlerUnfinishedPolicy = HandlerUnfinishedPolicy.WARN_AND_ABANDON,
```

Comment:
I wonder if, instead of this new overload, you can just add `description: Optional[str] = None` to the one above at line 223. Same for others, ideally you don't need a new overload for signal or update (you do for query because this is the first optional parameter overload)